website/docs/r/database_secret_backend_role.md

```diff
@@ -78,6 +78,40 @@ The following arguments are supported:
 * `max_ttl` - (Optional) The maximum number of seconds for leases for this
   role.
 
+* `credential_type` `(string: "password")` – Specifies the type of credential that
```

Comment:
nit: I like including the type information. But it looks a bit awkward with the existing documentation that does not have it. Should we either change the existing documentation to match this new format or align our new changes with the existing format?

Reply:
What do you mean with the existing documentation? To my knowledge, all of our documentation around the credential types should be consistent, I think?

Reply:
In this file, if you look at `max_ttl` and all the fields documented above `credential_type` you can see that they do not include the type information as you have done. This is a nit though so feel free to ignore if you don't agree.

Reply:
Ah got it! Yes, that makes perfect sense. I updated it to be consistent with the rest of the docs.